<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNetworkServerForm</class>
 <widget class="QFrame" name="CNetworkServerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="vl_NetworkServerForm">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tw_ServerForm">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_ServerGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_ServerGeneral">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QFrame" name="fr_ServerGeneral">
         <layout class="QFormLayout" name="fl_ServerGeneral">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_Name">
            <property name="toolTip">
             <string>name and description</string>
            </property>
            <property name="text">
             <string>Name/desc.:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="wi_NameDesc" native="true">
            <layout class="QHBoxLayout" name="hl_NameDesc" stretch="1,2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="le_Name">
               <property name="placeholderText">
                <string>server name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_Description">
               <property name="placeholderText">
                <string>server description</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_Ecosystem">
            <property name="text">
             <string>Eco./type:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="wi_EcoType" native="true">
            <layout class="QHBoxLayout" name="hl_EcoType">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="swift::gui::CEcosystemComboBox" name="cbp_Ecosystem"/>
             </item>
             <item>
              <widget class="QComboBox" name="cb_ServerType"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_Address">
            <property name="text">
             <string>Addr./port:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="wi_AddressPort" native="true">
            <layout class="QHBoxLayout" name="hl_AdressPort" stretch="2,1">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="le_Address">
               <property name="placeholderText">
                <string>e.g. &quot;server.foo.com&quot;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_Port">
               <property name="text">
                <string>6809</string>
               </property>
               <property name="maxLength">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lbl_RealName">
            <property name="text">
             <string>Real name:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="le_RealName">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>your name if required</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lbl_IdPassword">
            <property name="text">
             <string>Id/password:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QWidget" name="wi_IdPassword" native="true">
            <layout class="QHBoxLayout" name="hl_IdPassword">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="le_NetworkId">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>id</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="wi_Password" native="true">
               <layout class="QHBoxLayout" name="hl_Password">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="le_Password">
                  <property name="maxLength">
                   <number>32</number>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>password</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="tb_Unhide">
                  <property name="toolTip">
                   <string>unhide password</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/diagona/icons/diagona/icons/question-button.png</normaloff>:/diagona/icons/diagona/icons/question-button.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_ServerFsd">
      <attribute name="title">
       <string>FSD</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_ServerFsd">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="swift::gui::editors::CFsdSetupForm" name="form_ServerFsd">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::editors::CFsdSetupForm</class>
   <extends>QFrame</extends>
   <header>gui/editors/fsdsetupform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::CEcosystemComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/ecosystemcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tw_ServerForm</tabstop>
  <tabstop>le_Name</tabstop>
  <tabstop>le_Description</tabstop>
  <tabstop>cbp_Ecosystem</tabstop>
  <tabstop>cb_ServerType</tabstop>
  <tabstop>le_Address</tabstop>
  <tabstop>le_Port</tabstop>
  <tabstop>le_RealName</tabstop>
  <tabstop>le_NetworkId</tabstop>
  <tabstop>le_Password</tabstop>
  <tabstop>tb_Unhide</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
